p3/management/commands/accepted_talks.py

```diff
@@ -93,8 +93,9 @@ def handle(self, *args, **options):
                   talk.title.lower().startswith('keynote')):
                 #print ('found keynote: %r' % talk)
                 type = 'k'
-            elif admin_type == 'x':
-                # Don't list these placeholders for the schedule
+            elif admin_type in ('x', 'o', 'c', 'l', 'r', 's', 'e'):
```

Comment:
On 24.05.2016 17:55, Alexandre M. S. wrote:
>> @@ -93,8 +93,9 @@ def handle(self, *args, **options):
>>                    talk.title.lower().startswith('keynote')):
>>                  #print ('found keynote: %r' % talk)
>>                  type = 'k'
>> -            elif admin_type == 'x':
>> -                # Don't list these placeholders for the schedule
>> +            elif admin_type in ('x', 'o', 'c', 'l', 'r', 's', 'e'):
> 
> The thing is that for the booklet I need these talks as well.

Are you using the same script ?

I thought I was the only one using it for creating the session
list.

-- 
-- 
Marc-Andre Lemburg
http://www.malemburg.com/